src/library/export/coverartsaver.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <QImage>
+#include <QString>
+
+class CoverArtSaver {
+  public:
+    CoverArtSaver(const QString& fileName, const QImage& origCoverArt);
+    ~CoverArtSaver() = default;
+
+    bool saveCoverArt();
+
+  private:
+    QString m_fileName;
+    QImage m_origCoverArt;
```

Comment:
Why is this a stateful class? Can saveCoverArt() become a free fucntion in the anonymous namespace?